### PR TITLE
Github Actions tests for Llava Next and modify pretrain recipe to have language model path

### PR DESCRIPTION
**Pull request by yashaswikarnati**
# What does this PR do ?

Adding Github Actions tests for Llava Next

**Collection**: VLM

# Changelog 
- github actions CI test for llava next training.
- modify pretrain recipe to have pretrained language model path

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# GitHub Actions CI

The Jenkins CI system has been replaced by GitHub Actions self-hosted runners.

The GitHub Actions CI will run automatically when the "Run CICD" label is added to the PR.
To re-run CI remove and add the label again.
To run CI on an untrusted fork, a NeMo user with write access must first click "Approve and run".

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [x] Did you write any new necessary tests?
- [x] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [x] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

**Review by yaoyu-33 — DISMISSED** [review]

File: .github/workflows/cicd-main.yml [hunk 1]
```diff
@@ -4607,6 +4607,21 @@ jobs:
...
+    with:
+      RUNNER: self-hosted-azure
+      SCRIPT: |
+        python tests/collections/vlm/test_llava_next_train.py \
+        --devices=1 \
+        --max-steps=5 \
+        --experiment-dir=/tmp/nemo2_llava_next_results/${{ github.run_id }}
+
+      AFTER_SCRIPT: |
+        rm -rf /tmp/nemo2_llava_next_results
```
Comment (pablo-garay) [inline .github/workflows/cicd-main.yml:4624]: this is strictly not needed to do

File: .github/workflows/cicd-main.yml [hunk 2]
```diff
@@ -4607,6 +4607,21 @@ jobs:
       AFTER_SCRIPT: |
         rm -rf /tmp/nemo2_ckpt
```
Comment (pablo-garay) [inline .github/workflows/cicd-main.yml:4608]: this is strictly not needed to do

File: .github/workflows/cicd-main.yml [hunk 3]
```diff
@@ -4607,6 +4607,21 @@ jobs:
...
         rm -rf /tmp/nemo2_ptq_engine
```
Comment (pablo-garay) [inline .github/workflows/cicd-main.yml:4609]: this is strictly not needed to do

**Review by pablo-garay — DISMISSED** [review]

File: .github/workflows/cicd-main.yml [hunk 4]
```diff
@@ -4607,6 +4607,21 @@ jobs:
...
+  
+  L2_NeMo_2_LLAVA_NEXT_MOCK_TRAINING:
+    needs: [cicd-test-container-setup]
+    uses: ./.github/workflows/_test_template.yml
+    if: contains(fromJSON(needs.cicd-test-container-setup.outputs.test_to_run), 'L2_NeMo_2_LLAVA_NEXT_MOCK_TRAINING') || needs.cicd-test-container-setup.outputs.all == 'true'
+    with:
+      RUNNER: self-hosted-azure
```
Comment (ko3n1g) [inline .github/workflows/cicd-main.yml:4616]: This seems to be a single-gpu test, so please use the following instead:

```suggestion
      RUNNER: azure-gpu-vm-runner1-cpu
```

**Review by ko3n1g — CHANGES_REQUESTED** [review]

File: .github/workflows/cicd-main.yml [hunk 5]
```diff
@@ -4607,6 +4607,21 @@ jobs:
       AFTER_SCRIPT: |
         rm -rf /tmp/nemo2_ckpt
         rm -rf /tmp/nemo2_ptq_engine
+  
+  L2_NeMo_2_LLAVA_NEXT_MOCK_TRAINING:
+    needs: [cicd-test-container-setup]
+    uses: ./.github/workflows/_test_template.yml
+    if: contains(fromJSON(needs.cicd-test-container-setup.outputs.test_to_run), 'L2_NeMo_2_LLAVA_NEXT_MOCK_TRAINING') || needs.cicd-test-container-setup.outputs.all == 'true'
+    with:
+      RUNNER: azure-gpu-vm-runner1-cpu
```
Comment (ko3n1g) [inline .github/workflows/cicd-main.yml:4616]:
```suggestion
      RUNNER: self-hosted-azure-gpus-1
```

**Review by ko3n1g — APPROVED** [review]
please await the manual run

**Pull request merged**